cluster-install/install.sh

```diff
@@ -1970,9 +1971,9 @@ step_generate_trident_fresh_install_yaml() {
 
     logheader $__DEBUG "Generating Trident YAML files..."
 
-    # TODO ASTRACTL-32183: use _KUBERNETES_VERSION to choose the right bundle yaml (pre 1.25 or post 1.25)
+    # TODO ASTRACTL-32138: use _KUBERNETES_VERSION to choose the right bundle yaml (pre 1.25 or post 1.25)
     insert_into_file_after_pattern "$kustomization_file" "resources:" \
-        "- https://github.com/NetApp-Polaris/astra-deploy/trident-installer/deploy?ref=$__GIT_REF_TRIDENT"
+        "- https://github.com/NetApp-Polaris/astra-connector/trident-deploy-files-tmp?ref=$__GIT_REF_TRIDENT"
```

Comment:
Ajay ran into an issue where github prompted him to authenticate to the `astra-deploy` repository while running the script, breaking the Trident fresh install. 

This is likely because he never cloned/authenticated with the `astra-deploy` repo before or because he doesn't have access. Moving the deploy files to the `astra-connector`  repository should allow more developers to run the script without running into this issue since most people have dealt with the astra connector at some point.